src/shared/ui/Error.tsx

```
@@ -0,0 +1,17 @@
+import { Error as ErrorIcon } from '@/assets/icons'
+import { StatusContainer, StatusText } from '@/shared/styles/status'
+
+const Error = ({ isFullPage = false }: { isFullPage?: boolean }) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)